package.json

```diff
@@ -128,8 +127,7 @@
     "chai": "^4.3.4",
     "coffeescript": "^2.6.1",
     "cross-env": "^7.0.2",
-    "eslint": "^8.56.0",
-    "eslint-plugin-n": "^17.15.1",
+    "eslint": "^9.22.0",
```

Comment:
Just commenting here as a reminder that this PR updates to ESLint v9 (might otherwise get lost in all the other changes)

Reply:
Yeah, I was surprised that we hadn't already updated